wagtail/admin/templates/wagtailadmin/home/locked_pages.html

```diff
@@ -49,7 +49,12 @@ <h2 class="title-wrapper">{% trans "Your locked pages" %}</h2>
                                 </ul>
                             </td>
                             <td>{# Deliberately empty #}</td>
-                            <td valign="top"><div class="human-readable-date" title="{{ page.locked_at|date:"DATETIME_FORMAT" }}">{% blocktrans trimmed with time_period=page.locked_at|timesince_simple %}{{ time_period }}{% endblocktrans %}</div></td>
+                            <td valign="top">
+                                <button type="button" class="button-tooltip" data-tippy-content="{{ page.locked_at|date:"DATETIME_FORMAT" }}">
```

Comment:
we have to be careful with the class name here, we are saying `button-tooltip` but what the styles say are 'button that looks like a link'.

So I recommend we separate the JS behaviour (dictated by `data-tippy-content`) and the styling behaviour (dictated by the class). This is the recommended approach in our development docs https://docs.wagtail.org/en/latest/contributing/ui_guidelines.html#css-guidelines (note: the latest docs).

On top of that we should work towards the goal of BEM classes, even if not used everywhere yet.

So the action here would be 
- change class name to `button button--as-link` (or similar)
- update the styles accordingly


**Update** Another approach is to make this its own component - see the main comment for how this could work.

Reply:
I made a separate component. The `button` class has styling that I don't want on human-readable-date class.